src/main/java/com/labC/libraryCheckoutSystem/CartService.java

```diff
@@ -0,0 +1,13 @@
+package com.labC.libraryCheckoutSystem;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public interface CartService {
+
+    @Autowired
+    CartRepo cartRepo;
+
+    public String saveCart(String book,boolean bookStatus);
```

Comment:
This looks good, however I would think about how the book status is checked. Should the boolean be passed into the method as a parameter? This may not need to change depending on what the team decides.